docs/xxusejitserver.md

```diff
@@ -0,0 +1,45 @@
+<!--
+* Copyright (c) 2021, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]UseJITServer
+
+This option starts up OpenJ9 JVM in JITServer client mode.
+
+## Syntax
+
+        -XX:[+|-]UseJITServer
+
+| Setting                 | Effect | Default                                                                            |
+|-------------------------|--------|:----------------------------------------------------------------------------------:|
+|`-XX:+UseJITServer`           | Enable |                                                                                    |
+|`-XX:-UseJITServer`           | Disable| :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+## Explanation
+
+When you enable this option, the JVM will try to connect to a server and send it all JIT compilations requests.
```

Comment:
"When you enable this option" --> "When this option is enabled"

Reply:
The documentation guidelines recommend using "you" rather than passive voice but I agree that passive voice sounds better in this case.